### PR TITLE
add keywords in gumshoe,  cc #5422

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #5422 
@pvnr0082t please help me review it, thanks.

### CODE REVIEW

**Comment by pvnr0082t** [comment]
@kennynaoh sorry for late review. please rebase on the latest master branch.

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have rebased to the latest master branch.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

File: ajax/libs/gumshoe/package.json [hunk 1]
```diff
@@ -7,6 +7,12 @@
     "name": "Chris Ferdinandi",
     "url": "http://gomakethings.com"
   },
+  "keywords": [
+    "gumshoe",
+    "html",
+    "javascript",
+    "gulp"
```
Comment (PeterDaveHello) [inline ajax/libs/gumshoe/package.json]: Why `gulp`?

Comment (kennynaoh) [inline ajax/libs/gumshoe/package.json]: I guessed it is the important tool related with `gumshoe`.
But now I think it is just dependency about `gumshoe`.
So maybe I should remove it, right?
Thanks!

Comment (PeterDaveHello) [inline ajax/libs/gumshoe/package.json]: There is no need to add gulp, there are millions of libs using gulp.

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have removed gulp.
Please help me review it again.
Thanks

**Comment by PeterDaveHello** [comment]
There is no reason not to use a single line commit message here.

**Comment by pvnr0082t** [comment]
So @kennynaoh please modify commit message into single line.

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have modified commit message to single line.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Comment by PeterDaveHello** [comment]
Please explain the keywords you choose to use here.

**Comment by PeterDaveHello** [comment]
985ac9e4b78bc237d0a21eae50d447a9fa1fd673

**Pull request closed without merging**